### PR TITLE
few minor fixes and one new email format

### PR DESCRIPTION
**Pull request by ibnbajja**
Nice project man. Tested working in Kali.

### Updates:
* Patched to fix broken selectors and new LinkedIn login page. 
* Added ability to input password without using an argument (better for bash history)
* Added new email format: firstInitialLastName@domain.com
* Changes to support a single portable binary.
* Added version # (and totally just made it up)

### Issues: 
* This still has a bug if you enter the wrong password it will tell you you logged in, but then it will receive an selector error. You did not login. The error message is wrong. 
* An occasional issue on some hosts that likely indicates an issue with 2FA or possibly even captchas? Who knows.
* Might not work while being used a root user.

### CODE REVIEW

File: LinkedinEmails/Client.cs [hunk 1]
```diff
@@ -362,7 +363,7 @@ private static void SaveFile<T>(List<T> obj, string prefix)
 
             try
             {
-                File.WriteAllText(filename, JsonConvert.SerializeObject(obj, Formatting.Indented));
+                File.WriteAllText(filename, JsonConvert.SerializeObject(obj, Formatting.Indented), Encoding.UTF8);
```
Comment (ibnbajja) [inline LinkedinEmails/Client.cs:366]: Prevent failure when encountering an emoji.

**Comment by miltinhoc** [comment]
Thank you so much for helping out, its working nicely. I will try and focus on the issues you pointed out. I saw another one, sometimes before the "insert pin" page, there is another page that asks you if you want to use your linkedin app or pin to verify, so It also fails on these occasions. I will probably add a check for it

**Comment by miltinhoc** [comment]
> Thank you so much for helping out, its working nicely. I will try and focus on the issues you pointed out. I saw another one, sometimes before the "insert pin" page, there is another page that asks you if you want to use your linkedin app or pin to verify, so It also fails on these occasions. I will probably add a check for it

image

**Comment by ibnbajja** [comment]
> > Thank you so much for helping out, its working nicely. I will try and focus on the issues you pointed out. I saw another one, sometimes before the "insert pin" page, there is another page that asks you if you want to use your linkedin app or pin to verify, so It also fails on these occasions. I will probably add a check for it


Ahhhh I bet this was the random failure I was facing when doing testing that I thought may be 2FA or a captcha. Also the issues that I listed don't impact me personally. I was just listing them for full disclosure. 

Thanks for the merge. Cheers

**Pull request merged**